src/Components/BestSellers/BestSellers.jsx

```diff
@@ -0,0 +1,30 @@
+import './BestSellers.css'
+import BestSellerCard from "./BestSellerCard"
+import { v4 as uuid } from "uuid";
+
+const bestSellerData=[{
```

Comment:
you might add this dataset to backend folder maybe

Reply:
I'll do that Srishti, once I add data to the backend